homeassistant/components/stream/hls.py

```diff
@@ -109,14 +136,19 @@ def name(self) -> str:
     @property
     def format(self) -> str:
         """Return container format."""
-        return "mpegts"
+        return "mp4"
 
     @property
     def audio_codec(self) -> str:
         """Return desired audio codec."""
         return "aac"
 
     @property
-    def video_codec(self) -> str:
-        """Return desired video codec."""
-        return "h264"
+    def video_codecs(self) -> tuple:
+        """Return desired video codecs."""
+        return {"hevc", "h264"}
+
+    @property
+    def container_options(self) -> dict:
+        """Return container options."""
+        return {"movflags": "frag_custom+empty_moov+default_base_moof"}
```

Comment:
Added frag_custom since we are no longer cutting segments on every keyframe (default frags on every keyframe)